pgwidget/pgwidget_core.py

```diff
@@ -1420,9 +1421,12 @@ def on_unclick(self, glc):
         
 
 class Entry(TextContainerRect):
-    def __init__(self, text, pos, size, color=c.white, is_draggable=True, relative_pos=[0, 0],is_password=False,selection_color=c.red):
-        self.is_password=is_password #must be initialized before text definition (in super())
+    def __init__(self, text, pos, size, color=c.white, is_draggable=True, relative_pos=[0, 0],type='text',selection_color=c.red, file_types=None):
+        self.type=type #must be initialized before text definition (in super())
         super().__init__(text, pos, size, color=color, is_draggable=is_draggable, relative_pos=relative_pos,selection_color=selection_color)
+        if file_types is None:
+            file_types = []
+        self.file_types = file_types
```

Comment:
maybe rename to input_file_types + add docstring, default behaviour could accept any file type (maybe its working like that with empty list as you have it, I dont remember)

Reply:
i expect we can use this for outputting files as well. keeping file_types name

Reply:
replaced `[]` with explicit [('all files', '*')]